var/spack/repos/builtin/packages/gptune/package.py

```diff
@@ -48,6 +48,7 @@ class Gptune(CMakePackage):
     depends_on('py-requests', type=('build', 'run'))
     depends_on('py-cython', type=('build', 'run'))
     depends_on('py-pyaml', type=('build', 'run'))
+    depends_on('py-statsmodels@0.13.2:', type=('build', 'run'))
```

Comment:
I find it hard to believe that gptune 2.1.0 (released on Feb 2) requires statsmodels 0.13.2 (released on Feb 8). Is this a hard requirement? Does it only apply to master?

Reply:
I've changed this to depends_on('py-statsmodels@0.13.0:', type=('build', 'run')). Can you recheck this PR?